templates/repo/commits_list_group_by_date.tmpl

```diff
@@ -0,0 +1,113 @@
+{{$index := 0}}
+{{range .GroupCommits}}
+{{$index = Eval $index "+" 1}}
+<div class="ui timeline commits_list_group_by_date" data-index="{{$index}}">
+
+	<div class="timeline-badge-wrapper">
+		<div class="timeline-badge">
+			{{svg "octicon-git-commit"}}
+		</div>
+	</div>
+
+	<div class="timeline-body">
+		<h3 class="flex-text-block tw-py-2 timeline-heading">
+			Commits on {{DateUtils.AbsoluteShort .Date}}
```

Comment:
What if the browser's timezone is different from the server's? Will the formatted "date" be overall right or misleading users?

Reply:
I think it should be correct to use the server time as the standard currently, right? The `DateUtils.AbsoluteShort` is also used in multiple places across the entire system. I believe there won't be any issues with consistency.

Reply:
No. Other `AbsoluteShort` calls uses timestamp directly, so the timezone is always right there.

Reply:
OK, I've now changed the type of `Date` here to `timestamp` as well.